content/tutorial/tutorial.md

```diff
@@ -619,19 +615,19 @@ Each time a player moves, `xIsNext` (a boolean) will be flipped to determine whi
   }
 ```
 
-With this change, "X"s and "O"s can take turns. Try it!
+بها التغيير تستطيع `X` و `O` أخذ الأدوار. 
 
-Let's also change the "status" text in Board's `render` so that it displays which player has the next turn:
+فلنُغيِّر أيضًا نص الحالة في التابع `render` للمكوّن `Board` بحيث يعرض من هو اللاعب الذي سيلعب الدور التالي:
 
 ```javascript{2}
   render() {
-    const status = 'Next player: ' + (this.state.xIsNext ? 'X' : 'O');
+    const status = 'اللاعب التالي: ' + (this.state.xIsNext ? 'X' : 'O');
```

Comment:
This could be confusing since we didn't translate the screenshots from the game nor the inspect-element screens nor the codepen examples.

Our convention is to translate the code examples, but I think that we can drop this rule for now, and we comeback to this section by translating screenshots & codepens... So we just keep it like this for now and refere that "Next player" means "اللاعب التالي"
```suggestion
    const status = 'Next player: ' + (this.state.xIsNext ? 'X' : 'O');
```